src/__tests__/resetpassword/ResetPassword.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { ResetPassword } from "../../components/auth/ResetPassword";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("forgotPassword component", () => {
+  const dispatch = jest.fn();
+  const props = { dispatch, message: {} };
+
+  it("renders without crashing", () => {
+    expect(
+      shallow(<ResetPassword {...props} />)
+        .find("form.resetPassword")
+        .exists()
+    ).toBe(true);
+  });
+  it("renders an submit button", () => {
+    expect(shallow(<ResetPassword {...props} />).find("#submit").length).toBe(
+      1
+    );
+  });
+  describe("Component input fields", () => {
```

Comment:
You really hate empty lines. Empty line after each `it()`. Esp before a `describe()`

Reply:
No, I don't. Its my editor

Reply:
I don't hate them; they hate me